ros/lib/config.py

```diff
@@ -141,6 +141,7 @@ def build_endpoint_url(ep):
 DAYS_UNTIL_STALE = int(os.getenv("DAYS_UNTIL_STALE", '45'))
 CW_LOGGING_FORMAT = '%(asctime)s - %(levelname)s  - %(funcName)s - %(message)s'
 ROS_PROCESSOR_PORT = int(os.getenv("ROS_PROCESSOR_PORT", "8000"))
+ROS_PCP_PROCESSOR_PORT = int(os.getenv("ROS_PCP_PROCESSOR_PORT", "8000"))
```

Comment:
I would suggest to use same constant `ROS_PROCESSOR_PORT`. As part of new architecture, there will be need of introducing multiple processors so it would be good to avoid specific term.

Reply:
As discussed offline, we will keep this naming as it is to avoid confusion and conflicts. Before merging the things into the main branch, we can remove this specific naming that talks about `pcp`.